### ISSUE TITLE
Error while executing finsecbugs.sh on ubuntu

### ISSUE THREAD

**Issue by palok86** [find-sec-bugs/find-sec-bugs#367]
Hi,
I am trying to use findsecbugs on ubuntu but evertyime i am getting an exception.

My java variables are set but still getting the issue.
However, the same command works on Windows with the batch file.

```
./findsecbugs.sh "/home/vp/Desktop/Demo Proj/WebGoat-7.0.1/webgoat-container/target/*.class"
Exception in thread "main" java.io.IOException: IOException while scanning codebases
	at edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:285)
	at edu.umd.cs.findbugs.FindBugs.runMain(FindBugs.java:393)
	at edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1200)
	at edu.umd.cs.findbugs.LaunchAppropriateUI.launch(LaunchAppropriateUI.java:106)
	at edu.umd.cs.findbugs.LaunchAppropriateUI.main(LaunchAppropriateUI.java:198)
Caused by: edu.umd.cs.findbugs.classfile.ResourceNotFoundException: Resource not found: java/lang/Object.class
	at edu.umd.cs.findbugs.classfile.impl.ClassPathImpl.lookupResource(ClassPathImpl.java:164)
	at edu.umd.cs.findbugs.classfile.impl.ClassPathBuilder.build(ClassPathBuilder.java:282)
	at edu.umd.cs.findbugs.FindBugs2.buildClassPath(FindBugs2.java:677)
	at edu.umd.cs.findbugs.FindBugs2.execute(FindBugs2.java:218)
	... 4 more
```

**Comment by h3xstream** [comment]
I will repackage the latest spotbugs version in the next release. It should fix this issue. See #365

**Comment by andresriancho** [comment]
Same issue here with:
 * findsecbugs-cli-1.7.1.zip
 * Java(TM) SE Runtime Environment (build 1.8.0_171-b11)

**Comment by h3xstream** [comment]
This should be fix in #445

**Comment by h3xstream** [comment]
I will do a round of integration test on the next release. I totally ignored the Linux on the pass two releases.
